Cargo.toml

```diff
@@ -8,16 +8,16 @@ repository = "https://github.com/yeastplume/rust-sssmc39"
 description = "Work-in-Progress of Shamir Secret Sharing for Mnemonic Codes (SLIP-39)"
 
 [dependencies]
-lazy_static = "1"
-bitvec = "0.17"
-failure = "0.1"
-failure_derive = "0.1"
-rand = "0.6"
-hmac = "0.7"
-sha2 = "0.8"
-ring = { version = "0.16", optional = true }
-digest = { version = "0.8", optional = true }
-pbkdf2 = { version = "0.3", optional = true, default-features = false }
+lazy_static = "1.4.0"
+bitvec = "1.0.1"
```

Comment:
Pretty interesting that all the breaking changes in bitvec usage has left this codebase intact.